### PR TITLE
Update cloud-billing.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Update for the new Cloud Trial Experience

### CODE REVIEW

File: source/cloud/cloud-billing/cloud-billing.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Mattermost Cloud is offered as a pay-as-you-go subscription service.
 
 You will be billed for registered users in your workspace. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they are not deactivated in the billing period.
 
-Mattermost is available free of charge for all workspaces that have 10 or fewer registered users. Your subscription will be upgraded to a per-user pricing model once you reach 11 or more registered users.
+Customers who tried Mattermost Cloud from our October 2020 launch, can use Mattermostfree of charge for with 10 or fewer registered users and will be upgraded to a per-user pricing model once reaching 11 or more registered users. Now new workspaces created can trial Mattermost Cloud free for 14 days before upgrading to a per-user monthly subscription.
```
Comment (cwarnermm) [inline source/cloud/cloud-billing/cloud-billing.rst:8]:
```suggestion
Customers who signed up for a Mattermost Cloud trial from our October 2020 launch can continue to use their Cloud instance free of charge for up to ten registered users; however, these customers must upgrade to a per-user pricing model after reaching 11 or more registered users. New Mattermost Cloud workspaces can trial Mattermost Cloud free for 14 days before being required to upgrade to a monthly per-user subscription. 
```

File: source/overview/mattermost-cloud-overview.rst [hunk 2]
```diff
@@ -37,21 +37,21 @@ Frequently Asked Questions (FAQs)
 How much does Mattermost Cloud cost?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Mattermost Cloud Professional is free for up to 10 users. After 10 users, it is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users please contact us.
+Mattermost Cloud Professional is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users please contact us.
```
Comment (cwarnermm) [inline source/overview/mattermost-cloud-overview.rst:40]: @wiersgallak - Does this line need to change as well?

File: source/overview/mattermost-cloud-overview.rst [hunk 3]
```diff
@@ -37,21 +37,21 @@ Frequently Asked Questions (FAQs)
...
 
 Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.    
 
 How can customers trial Mattermost Cloud?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Mattermost Cloud Professional edition is free for up to ten users and all features are provided in this free offering.
+Mattermost Cloud Professional edition is available to trial for 14 days.
```
Comment (cwarnermm) [inline source/overview/mattermost-cloud-overview.rst:47]:
```suggestion
Mattermost Cloud Professional edition is available to try free for up to 14 days. 
```

**Review by cwarnermm — COMMENTED** [review]
Suggestions proposed inline

**Review by marianunez — APPROVED** [review]

File: source/cloud/cloud-billing/cloud-billing.rst (hunk 1 above)
Comment (wiersgallak) [reply source/cloud/cloud-billing/cloud-billing.rst:8]: @cwarnermm They weren't really trialing, it was a free offer.  Changed this slightly to remove the word trial you had suggested after "...who signed up for a Mattermost Cloud...." in the first sentence.

File: source/overview/mattermost-cloud-overview.rst (hunk 2 above)
Comment (wiersgallak) [reply source/overview/mattermost-cloud-overview.rst:40]: I chose to keep this to what will be released as the new process.

**Review by cwarnermm — APPROVED** [review]
Reads well!

**Pull request merged**